make_figures.py

```diff
@@ -203,24 +231,74 @@ def make_timeplot(df_measure, df_prediction, countries=None):
             )
     ))
 
-    fig.add_annotation(
-            x=0.1,
-            y=0.95,
-            xref='paper',
-            yref='paper',
-            showarrow=False,
-            font_size=LABEL_FONT_SIZE,
-            text="Confirmed cases per Million")
-    fig.add_annotation(
-            x=1,
-            y=-0.13,
-            xref='paper',
-            yref='paper',
-            showarrow=False,
-            font_size=LABEL_FONT_SIZE - 6,
-            font_color="DarkSlateGray",
-            text="Drag handles below to change time window",
-            align="right")
+    # fig.add_annotation(
+    #         x=0.1,
+    #         y=0.95,
+    #         xref='paper',
+    #         yref='paper',
+    #         showarrow=False,
+    #         font_size=LABEL_FONT_SIZE,
+    #         text="Confirmed cases per Million")
+    # fig.add_annotation(
+    #         x=1,
+    #         y=-0.13,
+    #         xref='paper',
+    #         yref='paper',
+    #         showarrow=False,
+    #         font_size=LABEL_FONT_SIZE - 6,
+    #         font_color="DarkSlateGray",
+    #         text="Drag handles below to change time window",
+    #         align="right")
+    fatalities_annotation = [dict(x=0.1,
+                                 y=0.95,
+                                 xref='paper',
+                                 yref='paper',
+                                 showarrow=False,
+                                 font_size=LABEL_FONT_SIZE,
+                                 text="fatalities cases per Million"
+                                 )]
+    confirmed_annotation = [dict(x=0.1,
+                                 y=0.95,
+                                 xref='paper',
+                                 yref='paper',
+                                 showarrow=False,
+                                 font_size=LABEL_FONT_SIZE,
+                                 text="Confirmed cases per Million"
+                                 )]
+    drag_handle_annotation = [dict(x=1,
+                                   y=-0.13,
+                                   xref='paper',
+                                   yref='paper',
+                                   showarrow=False,
+                                   font_size=LABEL_FONT_SIZE - 6,
+                                   font_color="DarkSlateGray",
+                                   text="Drag handles below to change time window",
+                                   align="right")]
+
+    visibility_tag_inv = list(np.logical_not(visibility_tag))
+    fig.update_layout(
+    updatemenus=[
```

Comment:
I think this erases the log/lin buttons, you would need to combine all of them in the `updatemenus` list.

Reply:
Yes, I saw that yesterday,  and its in the new updates that  I will push tonigt